### PR TITLE
docs: move variables to readme

### PR DESCRIPTION
**Pull request by lyricwulf**
makes it a bit nicer to read and configure

from this:
image

moved to readme:
image

future work: i would also like to recategorize these settings at some point. imo it would make sense for Idle - status / state / details to be grouped rather than status - idle / editing / viewing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -352,7 +352,7 @@
                     "vscord.status.image.small.viewing.text": {
                         "type": "string",
                         "default": "{app_name}",
-                        "description": "Custom string for the small image text when viewing\n\t- {app_name} will be replaced with the name of the app\n\t- {app_id} will be replaced with the id of the app"
+                        "description": "Custom string for the small image text when viewing- {app_name} will be replaced with the name of the app- {app_id} will be replaced with the id of the app"
```
Comment (xhayper) [inline package.json:355]: This line is uh

File: README.md [hunk 2]
```diff
@@ -40,11 +40,6 @@
             <img alt="Gitter" src="https://img.shields.io/badge/gitter-support chat-green?color=40aa8b" />
         </a>
     </p>
-    <p align="center">
-        <a href="https://github.com/leonardssh/vscord/actions/workflows/CD.yml">
-            <img alt="Continuous Delivery" src="https://github.com/leonardssh/vscord/actions/workflows/CD.yml/badge.svg" />
```
Comment (xhayper) [inline README.md:45]: why was this removed?

File: README.md [hunk 3]
```diff
@@ -89,6 +84,34 @@ Use the **[Extension Marketplace](https://code.visualstudio.com/docs/editor/exte
 
 ![a4](https://i.imgur.com/qMzox38.gif)
 
+## Variables
```
Comment (xhayper) [inline README.md:87]: Add an emoji to the name, maybe ❔ or ❓

**Review by xhayper — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (lyricwulf) [reply README.md]: good question

**Review by xhayper — APPROVED** [review]
